testing_utils/indicator_validation.ipynb

```diff
@@ -172,7 +172,130 @@
   },
   {
    "source": [
-    "## Qualitative Comparisons\n"
+    "## Qualitative Comparisons: Confirmed Incidence Prop"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.chdir(join(ROOT_DIR, \"jhu\"))\n",
+    "geocode = \"fips\"\n",
+    "signal_type = \"confirmed_incidence_prop\"\n",
+    "start_date = date(2020, 3, 1)\n",
+    "end_date = date(2020, 10, 1)\n",
+    "local_data = pd.concat(\n",
+    "        load_csvs(\n",
+    "            filter_filenames(\n",
+    "                read_relevant_date_filenames(\n",
+    "                    \"receiving\", \n",
+    "                    generate_date_range(\n",
+    "                        start_date.strftime(\"%Y%m%d\"), \n",
+    "                        end_date.strftime(\"%Y%m%d\")\n",
+    "                    )\n",
+    "                ),\n",
+    "                geocode,\n",
+    "                signal_type\n",
+    "            )\n",
+    "        )\n",
+    "    )\n",
+    "\n",
+    "remote_data_filename = join(ROOT_DIR, \"testing_utils\", ( \n",
+    "    start_date.strftime(\"%Y%m%d\") + \n",
+    "    \"_\" + \n",
+    "    end_date.strftime(\"%Y%m%d\") + \n",
+    "    \"jhu-csse_\" + \n",
+    "    \"confirmed_incidence_prop_\" + \n",
+    "    \"county.csv\"\n",
+    "))\n",
+    "if not isfile(remote_data_filename):\n",
+    "    remote_data = covidcast.signal(\"jhu-csse\", \"confirmed_incidence_prop\",\n",
+    "                            start_date, end_date,\n",
+    "                            \"county\")\n",
+    "    remote_data.to_csv(remote_data_filename)\n",
+    "else:\n",
+    "    remote_data = pd.read_csv(remote_data_filename)\n",
+    "remote_data[\"time_value\"] = pd.to_datetime(remote_data[\"time_value\"])\n",
+    "remote_data[\"geo_value\"] = remote_data[\"geo_value\"].astype(str)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Total difference:  2.3797075421327918e-12\n"
+     ]
+    },
+    {
+     "output_type": "display_data",
+     "data": {
+      "text/plain": "<Figure size 1152x432 with 1 Axes>",
```

Comment:
Should we be able to see the remote line or are they identical here?  I cannot see the remote line on any of the new charts.